src/main/kotlin/tech/thatgravyboat/skyblockpv/screens/PvTab.kt

```diff
@@ -34,6 +32,7 @@ enum class PvTab(
     FISHING(FishingScreen::class, Items.FISHING_ROD.defaultInstance),
     PETS(PetScreen::class, Items.BONE.defaultInstance),
     FARMING(BaseFarmingScreen::class, ::FarmingScreen, Items.WHEAT.defaultInstance),
+    CF(ChocolateFactoryScreen::class, SkullTextures.CF.createSkull())
```

Comment:
Same as above

Reply:
this one doesnt need caching as its used soon as you open the screen and only created once similarly to all thew defaultInstances above it